content/en/releases/platforms.md

```diff
@@ -0,0 +1,336 @@
+
+#### Extra files (for GCE)
+
+There are also corresponding `*.sha256` and `*.sha512` for every file below
+
+| File                        |
+| --------------------------- |
+| configure.sh                |
+| master.yaml                 |
+| node.yaml                   |
+| shutdown.sh                 |
+| windows/common.psm1         |
+| windows/configure.ps1       |
+| windows/install-ssh.psm1    |
+| windows/k8s-node-setup.psm1 |
+| windows/user-profile.psm1   |
+
+#### Tar archives
+
+There are also corresponding `*.sha256` and `*.sha512` for every file below
+
+| File                                       | darwin/amd64 | linux/386 | linux/amd64 | linux/arm64 | linux/ppc64le | linux/s390x | windows/386 | windows/amd64 | portable |
+| ------------------------------------------ | :----------: | :-------: | :---------: | :---------: | :-----------: | :---------: | :---------: | :-----------: | :------: |
+| kubernetes-client-**[system-arch]**.tar.gz |      ✅      |    ✅     |     ✅      |     ✅      |      ✅       |     ✅      |     ✅      |      ✅       |          |
+| kubernetes-node-**[system-arch]**.tar.gz   |              |           |     ✅      |     ✅      |      ✅       |     ✅      |             |      ✅       |          |
+| kubernetes-server-**[system-arch]**.tar.gz |              |           |     ✅      |     ✅      |      ✅       |     ✅      |             |               |          |
+| kubernetes-test-**[system-arch]**.tar.gz   |      ✅      |           |     ✅      |     ✅      |      ✅       |     ✅      |             |      ✅       |    ✅    |
+| kubernetes-manifests.tar.gz                |              |           |             |             |               |             |             |               |          |
+| kubernetes-test.tar.gz                     |              |           |             |             |               |             |             |               |          |
+| kubernetes-src.tar.gz                      |              |           |             |             |               |             |             |               |          |
+| kubernetes.tar.gz                          |              |           |             |             |               |             |             |               |          |
+
+#### Other (uncategorized) files
+
+| File                    |
+| ----------------------- |
+| release/latest-1.16.txt |
+| release/latest-1.txt    |
+| release/latest.txt      |
+
+Files in directory: **archive**
+
+Comment: It looks like in the directory: **archive** we can find:
+
+- copy of **anago** (script used for building artifacts)
+- shallow copy of kubernetes repository
```

Comment:
Let's verify this and remove the Comment prefix if it's still true. (I'm pretty sure there's no `anago` any longer)

Reply:
I removed that section entirely.